### ISSUE TITLE
RecentBlockhashes sysvar needs to carry the full BlockhashQueue

### ISSUE THREAD

**Issue by t-nelson** [solana-labs/solana#10065]
need this to be all 300 blockhashes. If not in the 150, you can stop resending retries, but you can't yet resign and send a new transaction. To resign, you need to be sure the leader's block isn't still in flight. The RPC node will accept transaction if its blockhash is anywhere in that full 300.

_Originally posted by @garious in https://github.com/solana-labs/solana/pull/8632#issuecomment-629002981_

cc/ @mvines @garious

**Comment by mvines** [comment]
Is this an alternative? https://github.com/solana-labs/solana/pull/10054#discussion_r425898951

**Comment by garious** [comment]
Yes, that's a good alternative.  I'd be okay with closing this issue by adding detailed documentation saying that transactions with blockhashes not present in that list may still be in flight, but that leader nodes will reject the transaction; stop retrying, but DO NOT resign and resend yet!

**Comment by t-nelson** [comment]
@mvines I think we should go ahead and do the bump when convenient. `MAX_PROCESSING_AGE` is a non-consensus soft-cap, so using it here is giving it more weight than it deserves

**Comment by ryoqun** [comment]
@t-nelson CC: according to recent talk with @aeyakovenko, we're going to align with 150, not 300 for more shorter tx expiration for less data retention.
